zappa/__init__.py

```diff
@@ -10,11 +11,12 @@ def running_in_docker() -> bool:
     - When docker is used allow usage of any python version
     """
     # https://stackoverflow.com/questions/63116419
-    running_in_docker_flag = os.getenv("ZAPPA_RUNNING_IN_DOCKER", "False").lower() in ("true", "1", "t")
+    running_in_docker_flag = strtobool(os.getenv("ZAPPA_RUNNING_IN_DOCKER", "False").lower())
```

Comment:
Sorry, my comment wasn't clear. 

My suggestion was to use the conditions used in `strtobool()`, expanding the conditions only, not use `strtobool()`.
Can you update this to:

```
running_in_docker_flag = os.getenv("ZAPPA_RUNNING_IN_DOCKER", "False").lower() in ('y', 'yes', 't', 'true', 'on', '1')
```